packages/earljs/test/matchers/ContainerWith.test.ts

```diff
@@ -35,19 +35,21 @@ describe('ContainerWith matcher', () => {
 
   describe('in expectation', () => {
     it('works', () => {
-      earlExpect([1, 2, 3]).toEqual(earlExpect.containerWith(3))
+      earlExpect([1, 2, 3]).toEqual(earlExpect.containerWith([3]))
```

Comment:
`containerWith ` should take arguments as ...args not an array of arguments. `make` has to be tweaked.